src/app/view/Login-view.js

```diff
@@ -0,0 +1,130 @@
+"use client"
+
+import { useState } from "react";
+import CheckLoginDetails from '../controller/Login-controller';
+import Form from 'react-bootstrap/Form';
+import { useNavigate } from "react-router-dom";
+import { Link } from "react-router-dom";
+
+export default function Login() {
+  let Navigate = useNavigate();
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [usernameEmpty, setUsernameEmpty] = useState(false);
+  const [passwordEmpty, setPasswordEmpty] = useState(false);
+  const [incorrectCredentials, setIncorrectCredentials] = useState(false);
+  const [isHovered, setIsHovered] = useState(false);
+
+  const handleMouseEnter = () => {
+    setIsHovered(true);
+  };
+
+  const handleMouseLeave = () => {
+    setIsHovered(false);
+  };
+
+  async function SubmitLoginDetails(){
+    if (!username.trim() || !password.trim()) {
+      setUsernameEmpty(!username.trim());
+      setPasswordEmpty(!password.trim());
+      return;
+    }
+
+    try {
+      let result = await CheckLoginDetails(username, password);
+      if (result === "invalid"){
+        setIncorrectCredentials(false);
+      } else if (result === "error"){
+        setIncorrectCredentials(true);
+        setUsernameEmpty(false);
+        setPasswordEmpty(false);
+      } else {
+        setIncorrectCredentials(false);
+        const token = result.token;
+        console.log(token);
+        if (username === "adminUsername") {
+          Navigate("/adminHomepage");
+        } else {
+          Navigate("/driverHomepage");
+        }
+      }
+    } catch(error){
+        alert(error);
+    }
+  }
+
+  return (
+    <body style={{ position: 'absolute', margin: 0, padding: 0, width: '100%', height: '100%' }}>
+      <div id="root" style={{ position: 'relative' }}>
+        <div id="headerbackground" style={{
+          position: 'relative',
```

Comment:
Please please don't use inline CSS styles. Please use React-Bootstrap like I've asked before, it comes with elements that have CSS predefined https://react-bootstrap.netlify.app/ and you can use bootstrap classnames to change looks